alf/utils/value_ops.py

```diff
@@ -160,6 +160,7 @@ def discounted_return(rewards, values, step_types, discounts, time_major=True):
     is_lasts = (step_types == StepType.LAST).to(dtype=torch.float32)
     is_lasts = common.expand_dims_as(is_lasts, values)
     discounts = common.expand_dims_as(discounts, values)
+    rewards = common.expand_dims_as(rewards, values)
```

Comment:
The doc also needs to be updated. We should change Line 140 to 

`values (Tensor): When it represents values, the shape is [T,B] (or [T]). When it represents quantiles of a distribution, the shape is [T, B, num_quantiles] (or [T, num_quantiles]).`

Reply:
Updated.